CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - `pyrgg.engines.erdos_reyni` module
 ### Changed
+- `logger` function format for `erdos_reyni_gilbert` changed
+- `log` function added
```

Comment:
1. `log` --> `save_log`
2. Move this line to `### Added` section

Reply:
fixed in [631408e](https://github.com/sepandhaghighi/pyrgg/pull/182/commits/631408ee88efb334f5c86e64e5262b13ccdc7549)